pkg/k8s/labels.go

```diff
@@ -57,8 +58,26 @@ const (
 	ProxyVersionAnnotation = "conduit.io/proxy-version"
 )
 
+var ownerLabels = map[string]string{
+	"deployment":            ProxyDeploymentLabel,
+	"replicationcontroller": ProxyReplicationControllerLabel,
+	"replicaset":            ProxyReplicaSetLabel,
+	"job":                   ProxyJobLabel,
+	"daemonset":             ProxyDaemonSetLabel,
+}
```

Comment:
for consistency with source labels, can you change these to the following:
```go
	"deployment":             ProxyDeploymentLabel,
	"replication_controller": ProxyReplicationControllerLabel,
	"replica_set":            ProxyReplicaSetLabel,
	"k8s_job":                ProxyJobLabel,
	"daemon_set":             ProxyDaemonSetLabel,
```

for reference:
https://github.com/runconduit/conduit/blob/28d5007cdf0dbd93d50e20aea3e60689ab830a31/doc/proxy-metrics.md#conduit-labels-added-at-collection-time

also, consider making these strings constants, as they also appear in tests.

Reply:
Good catch! I didn't realize that these were derived from the `conduit.io/proxy-*` labels and needed to match. This change was originally suggested in https://github.com/runconduit/conduit/issues/721#issuecomment-379428283, but I've opened #740 to track that separately. In the meantime I've updated to use the existing labels and have added a test.